src/doc/selected_layers.cpp

```diff
@@ -99,6 +99,24 @@ LayerList SelectedLayers::toBrowsableLayerList() const
   return output;
 }
 
+LayerList SelectedLayers::toAllTilemaps() const
+{
+  LayerList output;
+
+  if (empty())
+    return output;
+
+  ASSERT(*begin());
+  ASSERT((*begin())->sprite());
+
+  for (Layer* layer : (*begin())->sprite()->allTilemaps()) {
+    if (contains(layer))
+      output.push_back(layer);
+  }
```

Comment:
I think we could iterate the selected layers and ask if they are tilemap (`isTilemap`) instead of asking for `contains()` of all tilemaps in the sprite.

```suggestion
  for (auto layer : *this) {
    if (layer->isTilemap())
      output.push_back(layer);
  }
```